yb-voyager/cmd/exportData.go

```diff
@@ -124,13 +125,13 @@ func exportDataOffline() bool {
 		createExportDataDoneFlag()
 		dfd := datafile.Descriptor{ExportDir: exportDir}
 		dfd.Save()
-		os.Exit(0)
+		atexit.Exit(0)
 	}
 
 	if liveMigration || useDebezium {
 		err := debeziumExportData(ctx, finalTableList)
 		if err != nil {
-			utils.PrintAndLog("Failed to run live migration: %s", err)
+			utils.PrintAndLog("Debezium export failed: %s", err)
```

Comment:
Refrain from using the underlying tool's name in user facing error messages (tool name in the log messages is fine).
This message can be reworded as `"export data command failed with error: %s", err`

Reply:
Hmm, but we can't totally avoid this right? I mean somewhere down the line, the error will have context of debezium, right? 
Also, this PR makes a change so that if there is a debezium exception, it gets printed in console as debezium stderr, similar to how we do it for ora2pg now.

Reply:
Consider the error messages from the point of view of users. The words "debezium export" is not meaningful to the user.
The console will completely belong to the `ProgressTracker` to emit anything on the console. Do not output anything on the console to report export status/error. The stdout/stderr collected in `bytes.Buffer`s should be logged in the voyager log files.